lessons/python_packages/source-lesson.md

```diff
@@ -0,0 +1,217 @@
+# Creating Packages in Python
```

Comment:
That's correct. It was only in there for convenience while editing. I'll remove it, because it's cited at the start.